twake/backend/node/src/services/channels/entities/channel.ts

```diff
@@ -1,9 +1,10 @@
 import { ObjectID } from "mongodb";
+import Uuid from "cassandra-driver"
```

Comment:
No, we must not rely on cassandra-driver in this place. If we have to, this is because the node-uuid module does not provide what we need